src/contracts/interfaces/IPool.sol

```diff
@@ -185,13 +185,29 @@ interface IPool {
     uint256 variableBorrowIndex
   );
 
+  /**
+   * @dev Emitted when the deficit of a reserve is covered.
+   * @param reserve The address of the underlying asset of the reserve
+   * @param caller The caller that triggered the DeficitCovered event
+   * @param amountCovered The amount of deficit covered
+   */
+  event DeficitCovered(address indexed reserve, address caller, uint256 amountCovered);
+
   /**
    * @dev Emitted when the protocol treasury receives minted aTokens from the accrued interest.
    * @param reserve The address of the reserve
    * @param amountMinted The amount minted to the treasury
    */
   event MintedToTreasury(address indexed reserve, uint256 amountMinted);
 
+  /**
+   *
+   * @param user The user address where the bad debt will be burned
+   * @param debtAsset The address of the underlying borrowed asset to be burned
+   * @param amount The amount to burn
+   */
+  event DeficitCreated(address indexed user, address indexed debtAsset, uint256 amount);
```

Comment:
does it make sense to rename to `amountBurned`? feels more aligned with `DeficitCovered` and is more descriptive of what's going on